src/utils.js

```diff
@@ -41,30 +41,37 @@ export function stripHash(regex) {
 
 /**
  * @param {number} delta
- * @param {number} difference
+ * @param {number} originalSize
  */
-export function getDeltaText(delta, difference) {
+export function getDeltaText(delta, originalSize) {
 	let deltaText = (delta > 0 ? '+' : '') + prettyBytes(delta);
-	if (delta && Math.abs(delta) > 1) {
-		deltaText += ` (${Math.abs(difference)}%)`;
+	if (originalSize === 0) {
+		deltaText += ` (new file)`;
+	} else if (originalSize === -delta) {
+		deltaText += ` (removed)`;
+	} else if (Math.abs(delta) > 0) {
+		const percentage = Math.round((delta / originalSize) * 100);
+		deltaText += ` (${delta > 0 ? '+' : '-'}${Math.abs(percentage)}%)`;
 	}
 	return deltaText;
 }
 
 /**
  * @param {number} difference
  */
-export function iconForDifference(difference) {
-	let icon = '';
-	if (difference >= 50) icon = '🆘';
-	else if (difference >= 20) icon = '🚨';
-	else if (difference >= 10) icon = '⚠️';
-	else if (difference >= 5) icon = '🔍';
-	else if (difference <= -50) icon = '🏆';
-	else if (difference <= -20) icon = '🎉';
-	else if (difference <= -10) icon = '👏';
-	else if (difference <= -5) icon = '✅';
-	return icon;
+export function iconForDifference(delta, originalSize) {
+	if (originalSize === 0) return '🆘';
```

Comment:
Could you please explain what is this statement for ? If `originalSize` is 0 then it will show as 🆘 which doesn't seem correct to me.

Reply:
The original size is 0 when it's a new file, which is an infinite size increase for that file... So that's why it shows the SOS.

Reply:
I'd prefer a different icon for this:

```suggestion
	if (originalSize === 0) return '🆕';
```